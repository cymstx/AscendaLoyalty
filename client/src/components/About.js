import React from "react";
import NavBar from "./NavBar";

function About(){
    return (
        <div>
        <NavBar />
            This is the about page. Content coming up soon
        </div>
    );
}

export default About;
