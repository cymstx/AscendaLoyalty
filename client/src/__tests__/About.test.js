import React from 'react';
import '../matchMedia.js';
import About from '../components/routing/About';
import renderer from 'react-test-renderer'
import {BrowserRouter as Router} from 'react-router-dom';

describe("snapshot testing", () => {
    test("snapshot for Home component", () => {
        const renderedComponent = renderer.create(
        <Router>
            <About />
        </Router>).toJSON()
        expect(renderedComponent).toMatchSnapshot()
    })
}) 