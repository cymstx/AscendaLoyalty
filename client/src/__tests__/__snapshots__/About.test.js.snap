// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing snapshot for Home component 1`] = `
<div>
  This is the about page. Content coming up soon
</div>
`;
